### DISCUSSION TITLE
训练采样率 8k 的模型

### DISCUSSION THREAD

**Discussion by Jackwaterveg** [PaddlePaddle/PaddleSpeech#1980] [Ideas]
考虑到部分场景下，用户需要使用8k 的音频，那么提供8k 的模型就成为了一个重要的需求点。 关于 8k 的模型，目前可以参考 [callcenter](https://github.com/PaddlePaddle/PaddleSpeech/blob/develop/examples/callcenter/README.md)。

**Comment by harold-yh** [comment]
没有看到数据从哪里下载呢

**Reply by zh794390558** [reply]
只提供脚本，需要自己准别8k的数据。

**Reply by zh794390558** [reply]
also see https://github.com/PaddlePaddle/PaddleSpeech/discussions/2144
